keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -3049,6 +3051,11 @@
           "type": "string",
           "readOnly": true,
           "description": "The error message."
+        },
+        "innererror": {
```

Comment:
- Is this change also supported in the previous api-version? If yes, then should you also update the previous api-version specs?
- Are these the only changes for the new api-version i.e. is the change big enough to provision a new api-version (out of curiosity) ?